lib/anubis.go

```diff
@@ -218,6 +218,17 @@ func (s *Server) checkRules(w http.ResponseWriter, r *http.Request, cr policy.Ch
 		lg.Debug("rule hash", "hash", hash)
 		s.respondWithStatus(w, r, fmt.Sprintf("Access Denied: error code %s", hash), s.policy.StatusCodes.Deny)
 		return true
+	case config.RuleDenyAndReroute:
+		s.ClearCookie(w, s.cookieName, cookiePath)
+		lg.Info("deny and reroute", "reroute_to", cr.RerouteTo)
+		if cr.RerouteTo == nil || *cr.RerouteTo == "" {
+			lg.Error("reroute URL is missing for DENY_AND_REROUTE action")
+			s.respondWithError(w, r, "Internal Server Error: administrator has misconfigured Anubis. Please contact the administrator and ask them to look for the logs around \"maybeReverseProxy.RuleDenyAndReroute\"")
+			return true
+		}
+		// note for others, would it be better to be reverse proxying here?
```

Comment:
I just had some concerns related to anubis performance + how constantly the scrapers would end up this path